docs/faq.md

```diff
@@ -0,0 +1,204 @@
+with him rather than just building "another" JSON.NET. We want a strong
+ecosystem for .NET, but this can only happen if we embrace libraries based on
+merit, rather than by who wrote it. That's what open source is all about.
+
+## Why do you include APIs that don't work everywhere?
+
+We generally don't include APIs in .NET Standard that don't work everywhere, and
+instead provide them as libraries that sit above .NET Standard.
+
+But if you think about it: we can't make *type members* (e.g. methods,
+properties, etc) additive. The only thing you can make additive are *types*, as
+two different types can live in separate assemblies but we don't have a
+mechanism to split a single type across two different assemblies. In those
+cases, we leave the members on the type and let platforms that cannot
+meaningfully implement them throw `PlatformNotSupportedException`.
+
+Moving forward, we try to avoid creating types where only parts of it work
+everywhere. But as always, there will be cases where we couldn't predict the
+future and are forced to throw.
+
+## Will there be tooling to highlight APIs that don't work everywhere?
+
+Our current focus is on providing APIs either as part of .NET Standard or as
+independent packages that sit on top of .NET Standard. In some cases, certain
+APIs will not be supported everywhere and throw `PlatformNotSupportedException`.
+While that isn't ideal, it's much simpler than the alternatives, which are:
+
+* Using `#if`, i.e. cross-compile
+* Write complicated reflection code, i.e. runtime light-up
+
+A simple `if` statement with a platform check is much easier to express. Of
+course, there are limits to this. Exceptions are only acceptable for corner
+cases to avoid the complexities above. We will generally not expose large set of
+APIs that aren't supported.
+
+In the future, my hope is that we can provide tooling to help you with this, by,
+for example, providing Roslyn analyzers that can give you squiggles in the IDE.
+
+## Will Unity implement .NET Standard?
+
+Yes. We're working with Unity to make sure this is a smooth experience. In
+general, since Unity is a fork of Mono and it will mostly get .NET Standard
+support for free. The work to support Unity is mostly in tooling.
+
+## I saw your video and I like your watch. What is it?
+
+It's a Garmin Forerunner 920, a triathlon watch. Sadly, my watch is fitter than
+I am :-)
+
+## When will Visual Studio support creating .NET Standard libraries?
+
+The upcoming version of Visual Studio ("Dev15") will support the creation of
+.NET Standard Libraries, including building NuGet packages.
+
+## When will Xamarin Studio support creating .NET Standard libraries?
+
+The upcoming version of Xamarin Studio will support this.
+
+## What's the difference between .NET Standard Library and .NET Platform Standard?
+
+These were terms we used in earlier discussions. We only have one concept today
+called *.NET Standard*.
```

Comment:
What about the `NETStandard.Library` nuget package people see in project.json today? Will this go away?